simphony_mayavi/tests/testing_utils.py

```diff
@@ -11,6 +12,34 @@
 from simphony_mayavi.core.doc_utils import mergedocs
 
 
+def is_mayavi_older(version):
```

Comment:
I'd say leave the is_mayavi_older function, it's very convenient, but replace the internals with distutils.